src/Symfony/Bridge/Doctrine/SchemaListener/PdoCacheAdapterDoctrineSchemaSubscriber.php

```diff
@@ -16,8 +16,6 @@
 use Doctrine\ORM\Tools\ToolEvents;
 use Symfony\Component\Cache\Adapter\PdoAdapter;
 
-trigger_deprecation('symfony/doctrine-bridge', '5.4', 'The "%s" class is deprecated, use "%s" instead.', PdoCacheAdapterDoctrineSchemaSubscriber::class, DoctrineDbalCacheAdapterSchemaSubscriber::class);
```

Comment:
should this be moved to the constructor?

Reply:
If a project uses a `PdoAdapter` and `DoctrineBundle`, this class would be instanced a DBAL connection could be used. That's why I choose to remove the trigger.

Reply:
Can you think of a way to trigger a runtime deprecation notice somewhere?
We really try to never rely only on annotations.

Reply:
There is already a deprecation message when the `PdoAdapter` is instanced with a DBAL connection. And this class accepts only `PdoAdapter`.

If a trigger is really required, we can use reflection to check the value of the private property `PdoAdapter::$dbalAdapter`.

Reply:
Since we cannot be sure of what is injected in the constructor, a deprecation is triggered only when something different from `PdoAdapter` is used. See https://github.com/symfony/symfony/pull/44065/commits/4f10486ae81a226b6bad016d558ff08c1fad547d

Additionnaly, I added a check in `DoctrineDbalCacheAdapterSchemaSubscriberTest` to be sure we don't have to deal with incorrect usages of the class in the future.